### ISSUE TITLE
[SPRINT] ELAsTiCC - Run simulations of all classes for challenge

### ISSUE THREAD

**Issue by gnarayan** [LSSTDESC/SprintWeek2021#2]
# ELAsTiCC - Run simulations of all classes for challenge

We're ready to finalize SNANA input files and generate the simulations + serialize them into AVRO for all alerts. 
For efficiency, it helps to have a few (4-6) people submitting jobs in parallel, divvying up different classes.  

Contacts: Gautham, Renee, Rick
Day/Time: Full week
Main communication channel: #plasticc-public on slack
GitHub repo: https://github.com/LSSTDESC/plasticc_alerts

## Goals and deliverable
- Simulate DDF alerts and WFD alerts with Fast 10 option
- Make plots and validate that output looks reasonable
- Simulate full WFD for ~3 years
- Convert to AVRO and tarball
- Side goal is to train people who haven't run SNANA much - run at least one bootcamp on day 1

## Resources and skills needed
Minimally python - opening and reading FITS/txt files, plotting time series data, making histograms etc
Ideally a little familiarity with SNANA but we will do a tutorial session on day 1 on how to run these sims (will be recorded)
Will need NERSC accounts on Cori with SN environment setup

## Detailed description
Additional gotcha:
There are ~16 models but some models describe the same class of astrophysical objects - will need to combine the output of these simulations in some ratio.
Fun stretch goal can be updating the plasticc-kit from the original challenge or making this data available in a different form - e.g. sonification, or all sky map video
